workflow/workflow-demos/processes/AgileBPM/AgileBPM.p.json

```diff
@@ -344,9 +344,9 @@
           "name" : "send Mail",
           "config" : {
             "headers" : {
-              "emailTo" : "<%=ivy.case.getCreatorUser().getEMailAddress()%>",
+              "emailSubject" : "<%=in.caseInfo.kind%> <%=in.caseInfo.subject%>",
               "emailFrom" : "noreply@ivyserver.local",
-              "emailSubject" : "<%=in.caseInfo.kind%> <%=in.caseInfo.subject%>"
+              "emailTo" : "<%=ivy.case.getCreatorUser().getEMailAddress()%>"

```

Comment:
One of the bugs you are fixing?

Reply:
yes, this one here -> https://axonivy.atlassian.net/browse/XIVY-8841
hopefully we have seen this for the last time, as now we have set our 'expected' order. and will write it in the same manor on sub-sequent writes.